db-check.txt

```diff
@@ -0,0 +1,24 @@
+You are now connected to database "nc_news_test" as user "alice".
+ article_id |                         title                          | topic |    author     |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                body                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |     created_at      | votes 
```

Comment:
Probably dont need this file as part of the PR

Reply:
Could set up .gitignore to ignore these

Reply:
added to .gitignore